symtabAPI/src/dwarfWalker.C

```diff
@@ -360,11 +360,20 @@ bool DwarfWalker::parse_int(Dwarf_Die e, bool parseSib, bool dissociate_context)
         setMangledName(false);
 
         Dwarf * desc = dwarf_cu_getdwarf(e.cu);
-        dwarf_printf("(0x%lx) Parsing entry with context size %d, func %s, encl %p, (%s), mod:%s, abbr: %d, tag: %x\n",
+        if (!common_debug_initialized || common_debug_dwarf) {
```

Comment:
Do we really care about the abbreviation code *only if* we are debugging DWARF parsing?

Reply:
I was just fixing the code by maintaining its original behavior. This is a better question for @sashanicolas .

Reply:
we can actually just remove the call to "dwarf_getabbrevcode(e.abbrev)". It was necessary for debug/investigating only. Not needed anymore.

Reply:
@sashanicolas So we can remove the entire block from 363 to 376?

Reply:
Just remove the call parameter.
```
diff --git a/symtabAPI/src/dwarfWalker.C b/symtabAPI/src/dwarfWalker.C
index 1c9e583b2..92ae91efc 100644
--- a/symtabAPI/src/dwarfWalker.C
+++ b/symtabAPI/src/dwarfWalker.C
@@ -360,11 +360,11 @@ bool DwarfWalker::parse_int(Dwarf_Die e, bool parseSib, bool dissociate_context)
         setMangledName(false);
 
         Dwarf * desc = dwarf_cu_getdwarf(e.cu);
-        dwarf_printf("(0x%lx) Parsing entry with context size %d, func %s, encl %p, (%s), mod:%s, abbr: %d, tag: %x\n",
+        dwarf_printf("(0x%lx) Parsing entry with context size %d, func %s, encl %p, (%s), mod:%s, tag: %x\n",
                 id(), stack_size(),
                 curFunc()?curFunc()->getName().c_str():"(N/A)",
                 curEnclosure().get(), (dbg()!=desc)?"sup":"not sup",
-                mod()->fullName().c_str(), dwarf_getabbrevcode(e.abbrev), dwarf_tag(&e));
+                mod()->fullName().c_str(), dwarf_tag(&e));
 
         bool ret = false;
         switch(dwarf_tag(&e)) {
```